seq2science/workflows/scrna_seq/Snakefile

```diff
@@ -22,9 +22,10 @@ rule seq2science:
     Make a QC report and map
     """
     input:
-        expand([f"{{result_dir}}/{{quantifier}}/{custom_assembly(treps.loc[trep, 'assembly'])}-{trep}/run_info.json" for trep in treps.index] +
-               ([f"{{result_dir}}/seurat/{{quantifier}}/{custom_assembly(treps.loc[trep, 'assembly'])}-{trep}_seu_obj.RData" for trep in treps.index] + 
-               ["{result_dir}/seurat/kallistobus/{assemblies}_seu_merged.RData"] if config["export_seu_objects"] else []) + 
+        expand(([f"{{result_dir}}/{{quantifier}}/{custom_assembly(treps.loc[trep, 'assembly'])}-{trep}/run_info.json" for trep in treps.index] if config["quantifier"] == "kallistobus" else [] +
+               [f"{{result_dir}}/{{quantifier}}/{custom_assembly(treps.loc[trep, 'assembly'])}-{trep}/_tag_counts/run_report.yaml" for trep in treps.index] if config["quantifier"] == "citeseqcount" else []) +
```

Comment:
is there a check in the configuration already that makes sure the setting is clear?
```
if get_workflow() == "scrna_seq":
    assert config["quantifier"] in ["kallistobus", "citeseqcount"]
```

I think you raise an error in trimming.smk (around line 100), but that one seems like it's supposed to cover weird behaviour broadly(?)

Reply:
Good point, I think that needs to be fleshed out a bit more

Reply:
Just a general FYI. I just updated so that no exceptions and assertions are used anymore. It's easier on the eyes (especially for new users) if we log an error and then exit with an error code.

sth like this:

```
if quant not in []:
        logger.error(f"error message"
                  "Please make an issue on github if this is unexpected behaviour!")
    sys.exit(1)
```

Reply:
Added quantifier check